src/redux/actions/concepts/dictionaryConcepts.js

```diff
@@ -108,18 +108,18 @@ export const fetchDictionaryConcepts = (
   page = 1,
 ) => async (dispatch, getState) => {
   dispatch(isFetching(true));
-  let url = `${conceptType}/${conceptOwner}/sources/${conceptName}/concepts/?q=${query}&limit=${limit}&page=${page}&verbose=true`;
+  let url = `${conceptType}/${conceptOwner}/collections/${conceptName}/concepts/?q=${query}&limit=${limit}&page=${page}&verbose=true`;
```

Comment:
Won't this change prevent the latest version from being shown?

Reply:
I found out that only custom concepts were being shown when one uses sources instead of collections. I don't know if it affects changes made to a concept.

Reply:
@dkayiwa I have tested this and found that when using `/sources/` it shows the latest version of a dictionary concept while but only shows custom concepts while using `/collections/` shows all the concepts but doesn't show the latest concept version.

Reply:
My suggestion would be to use `/collections/` for now as it able to show all dictionary concepts. For example, if one was to add a CIEL concept at the moment they are not able to see the added concept(s). Later on, I would suggest the fixing of the `/collections/` endpoint to show the latest version of a concept or fixing the `/sources/` endpoint to also show other dictionary concepts.

Reply:
Do you think you can create a ticket for that?

Reply:
Yes, I can create a ticket for this.